### PR TITLE
feat(vision/v2): Release breaking changes as v2 module

### PR DESCRIPTION
**Pull request by quartzmo**
BREAKING CHANGE: removed fields from SafeSearchAnnotation in googleapis/go-genproto#809

### CODE REVIEW

File: vision/v2/go.mod [hunk 1]
```diff
@@ -0,0 +1,12 @@
+module cloud.google.com/go/vision/v2
+
+go 1.16
+
+require (
+	cloud.google.com/go v0.100.2
+	github.com/googleapis/gax-go/v2 v2.4.0
+	google.golang.org/api v0.78.0
+	google.golang.org/genproto v0.0.0-20220519153652-3a47de7e79bd
```
Comment (quartzmo) [inline vision/v2/go.mod:9]: @codyoss Is this genproto version OK?

Comment (codyoss) [reply vision/v2/go.mod:9]: LGTM

File: vision/v2/README.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+# Cloud Vision API
+
+[![Go Reference](https://pkg.go.dev/badge/cloud.google.com/go/vision.svg)](https://pkg.go.dev/cloud.google.com/go/vision)
+
+Go Client Library for Cloud Vision API.
+
+## Install
+
+```bash
+go get cloud.google.com/go/vision
```
Comment (codyoss) [inline vision/v2/README.md:10]: this should have /v2 at the end. Prolly missed this in the other PR.

**Review by codyoss — APPROVED** [review]
LGTM, just the one nit to fix

**Comment by quartzmo** [comment]
@codyoss PTAL at my test fix, thanks.

**Comment by codyoss** [comment]
@quartzmo Why did we have to make an exception for this v2 and not the last one with that one test?

**Comment by qnighy** [comment]
Is it intentional that some files (`apiv1/client.go`, `apiv1/image.go`, etc.) are missing from the v2 directory? Is there documentation on how to migrate these APIs to v2?

**Pull request merged**